climada/hazard/tc_tracks_synth.py

```diff
@@ -227,8 +234,9 @@ def calc_perturbed_trajectories(tracks,
                 except ValueError:
                     LOGGER.info('No land decay coefficients could be applied.')
             else:
-                LOGGER.error('No historical tracks contained. '
-                             'Historical tracks are needed for land decay.')
+                LOGGER.error('No historical tracks found. Historical'
+                             'tracks are needed for land decay calibration'
+                             'if use_global_decay_params=True.')
```

Comment:
Please, do not use the LOGGER for errors. Raise a proper error with e.g. 

`raise ValueError('')`. 
```suggestion
               raise ValueError('No historical tracks found. Historical'
                             'tracks are needed for land decay calibration'
                             'if use_global_decay_params=True.')
```